lxd/scriptlet/instance_placement.go

```diff
@@ -221,7 +221,12 @@ func InstancePlacementRun(ctx context.Context, l logger.Logger, s *state.State,
 
 			// Apply VM root disk size defaults if not specified.
 			if req.Type == api.InstanceTypeVM && rootDiskSizeStr == "" {
-				rootDiskSizeStr = storageDrivers.DefaultBlockSize
+				driver, err := storageDrivers.Load(s, rootDiskConfig["pool"], "", nil, nil, nil, nil)
```

Comment:
please can you add a function like `storageDrivers.DefaultVMBlockFilesystemSize` for this.